specification/migrate/resource-manager/Microsoft.OffAzure/stable/2020-01-01/migrate.json

```diff
@@ -0,0 +1,4910 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1",
```

Comment:
This should be set to `2020-01-01` and should be ideally caught by a linter rule.